packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -31,6 +31,9 @@
 )
 
 
+CLIENT_CREATION_ERROR_MESSAGE = "The measurement plug-in client creation for the service class '{}' has failed. Possible reason(s): {}"
```

Comment:
Copy the 1st sentence of this because it's more concise and readable: https://github.com/ni/measurement-plugin-python/blob/b9efedf4df35ef75a557fc91446f44c246d66cb6/packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py#L338

> f"Client creation failed for '{service_class}'.\nEither provide a module name or update the measurement with a valid service class."